__tests__/components/cards/wallet/CashoutAddress.test.tsx

```diff
@@ -0,0 +1,110 @@
+import "@testing-library/jest-dom";
+import { screen, fireEvent } from "@testing-library/react";
+import { Wallet } from "@/types/wallet";
+import { renderWithRedux } from "@__mocks__/renderWithRedux";
+import CashoutAddress from "@/components/cards/wallet/CashoutAddress";
+import { mockUser } from "@__mocks__/fixtures/user";
+import { mockWallet } from "@__mocks__/fixtures/wallet";
+import { KYCSTATUS, openVerificationModal } from "@/store/feature/kyc.slice";
+import { useDispatch } from "@/hooks/useTypedDispatch";
+
+jest.mock("@/hooks/useTypedDispatch.ts", () => ({
+  useDispatch: jest.fn(),
+}));
+
+const walletMock: Wallet = {
+  ...mockWallet,
+  title: "Test Wallet",
+  token: "BTC",
+  balance: 100,
+  description: "Test Wallet Description",
+  payouts: [],
+};
+
+jest.mock("@/store/feature/kyc.slice", () => {
+  const actual = jest.requireActual("@/store/feature/kyc.slice");
+  return {
+    ...actual,
+    openVerificationModal: jest.fn(),
+  };
+});
+
+const mockUserStore = {
```

Comment:
This is a fixtured instead: 
```suggestion
const FixtureUserStore = {
```